changelog/6189.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add a function `sunpy.map.extract_along_coord` that, for a given coordinate,
```

Comment:
```suggestion
Add a function `sunpy.map.extract_along_coord` that, for a given set of coordinates,
```
I think it makes more sense to talk about more than one coordinate here? (and adjust the other changelog lines appropriately)